R/check.R

```diff
@@ -89,6 +89,8 @@ revdep_env_vars <- function(force_suggests = FALSE) {
     "_R_CHECK_FORCE_SUGGESTS_" = as.character(force_suggests),
     "RGL_USE_NULL" = "TRUE",
     DISPLAY = "",
+    RSTUDIO = 0,
```

Comment:
Why would we set this to zero?

Reply:
Or this is for the subprocesses?

Reply:
This is for subprocesses which might pick up this settings. I'm getting unstable results from RStudio, running with this PR now. We'll see if it helps.